src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      calculator.result += number;
```

Comment:
let's use this instead of calculator all over the code, that would be more consistent (not all the time there will be a variable such as calculator so we have to learn how to handle such situations properly)
you'll hit into the issues with context loss, so here is the relevant article on this problem
https://www.freecodecamp.org/news/what-to-do-when-this-loses-context-f09664af076f/

Reply:
Намагався зробити через bind(calculator), але не вийшло. Зробив через декоратор, хоча мені здається що не до кінця зрозумів як воно працює. А як має виглядати рішення через bind?

Reply:
Дякую)